### DISCUSSION TITLE
Multiple accounts and WebHooks

### DISCUSSION THREAD

**Discussion by microshare-zz** [abraunegg/onedrive#1848] [Q&A] [answered]
Great work by the way. I appreciate all your hard work.

I have a requirement for multiple onedrive accounts across different onedrive/sharepoint tenants. 
Do I need separate webhooks url's or ports for each account or is that handled internally?

**Comment by abraunegg** [comment, marked as answer]
@microshare-zz 
> I have a requirement for multiple onedrive accounts across different onedrive/sharepoint tenants.
> Do I need separate webhooks url's or ports for each account or is that handled internally?

For *each* OneDrive and/or SharePoint configuration you require, you need a separate configuration for *each* configuration you desire. As such, additionally you will require a separate webhook URL / port / configuration for *each* configuration that you create.

**Reply by microshare-zz** [reply]
Thank you.
